.vsts-ci.yml

```diff
@@ -346,8 +348,12 @@ stages:
             _TestArg: ''
 
   - template: /eng/common/templates/jobs/source-build.yml
+    parameters:
+      disableComponentGovernance: true
 
   - template: /src/SourceBuild/Arcade/eng/common/templates/job/source-build-create-tarball.yml
+    parameters:
```

Comment:
This parameter doesn't exist.  This job doesn't invoke CG that I am aware of today.

Reply:
The pipelines report CG running on that job:
https://dev.azure.com/dnceng/internal/_build/results?buildId=1637460&view=results
![image](https://user-images.githubusercontent.com/12663534/156438230-e6ee9cc5-a5ff-4324-9cb3-faf65d648f0d.png)

Reply:
I see it is auto-injected.  I don't understand however how this non-declared parameter gets consumed and affects CG.